### PR TITLE
Avoid exceptions while getting agent config values when couch is down

### PR DESCRIPTION
**Pull request by amaltaro**
AgentStatusPoller component was complaining about those keys during a testbed intervention.

### CODE REVIEW

**Comment by amaltaro** [comment]
test this please

**Comment by ericvaandering** [comment]
test this please

**Comment by amaltaro** [comment]
It's good to go.

**Review by ticoann — APPROVED** [review]

File: src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py [hunk 1]
```diff
@@ -172,10 +172,8 @@ def updateCampaignConfig(self, campaignName, propDict):
         # update config DB
         resp = self.updateRecords('campaignconfig/%s' % campaignName, propDict)
 
-        if len(resp) == 1 and resp[0].get("ok", False):
-            return True
-        else:
-            return False
+        return bool(len(resp) == 1 and resp[0].get("ok", False))
```
Comment (ticoann) [inline src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py]: Do we need type casting to bool?

Comment (amaltaro) [reply src/python/WMCore/Services/ReqMgrAux/ReqMgrAux.py]: Hmm, apparently not :)

**Pull request merged**